src/main/scala/com/typesafe/sbt/packager/archetypes/systemloader/SystemloaderPlugin.scala

```diff
@@ -158,6 +162,7 @@ object SystemloaderPlugin extends AutoPlugin {
       "stop_runlevels" -> stopRunlevels.getOrElse(""),
       "start_facilities" -> startOn.getOrElse(""),
       "stop_facilities" -> stopOn.getOrElse(""),
+      "file_descriptor_limit" -> fileDescriptorLimit.getOrElse(""),
```

Comment:
What happens if `ulimit` is called with no arguments? Will this result in `unlimited`?